modules/mobile/config/routes.rb

```diff
@@ -5,8 +5,11 @@
 
   namespace :v0 do
     get '/user', to: 'users#show'
+    get '/letters', to: 'letters#letters'
```

Comment:
Rails convention would name this action `letters#index`

Reply:
fixed